app/main/posts/views/post-view-map.directive.js

```diff
@@ -94,12 +98,35 @@ function PostViewMap(PostEndpoint, Maps, _, PostFilters, L, $q, $rootScope, $com
             loadPosts().then(addPostsToMap);
         }
 
-        function loadPosts(query) {
+        function loadPosts(query, offset) {
             query = query || PostFilters.getQueryParams($scope.filters);
-
+            query.has_location = 'mapped';
+            offset = offset || 0;
+            var limit = 200,
```

Comment:
Could we hoist this to the top of the directive? ie. so its easy to find and tweak the magic number later when we need to.